tools/constava/constava.xml

```diff
@@ -0,0 +1,124 @@
+<tool id="constava" name="Constava" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="23.00">
+    <description>Conformational state variability from protein ensembles</description>
+    <macros>
+        <token name="@TOOL_VERSION@">1.1.0</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">constava</requirement>
+    </requirements>
+    <command detect_errors="exit_code">
+    <![CDATA[
+    #if $input_options.input_degrees == "Degrees"
+        #set angle_units = "--degrees"
+    #else
+        #set angle_units = ""
+    #end if
+
+    #if $pdf.use_pretrained_pdf
+        #set model_file = "custom_model.pkl"
+        #if $pdf.model_type == "kde"
+            constava fit-model -v --kde-bandwidth $pdf.bandwidth $angle_units -i $pdf.train_data -o $model_file
+        #else if $pdf.model_type == "grid"
+            constava fit-model -v --grid-points $pdf.grid_points --kde-bandwidth $pdf.bandwidth $angle_units -i $pdf.train_data -o $model_file
+        #end if
+        constava analyze -v -i '$input_files' -o 'output_constava.csv' --load-model $model_file --window $subsampling_options.window_size $angle_units
+    #else
+        constava analyze -v -i '$input_files' -o 'output_constava.csv' --window $subsampling_options.window_size $angle_units
+    #end if
+    ]]>
+    </command>
+    <inputs>
+        <section name="input_options" title="Input Options" expanded="true">
+            <param type="data" name="input_files" format="csv,xvg" label="Dihedral angles file" help="Upload the CSV or XVG file containing dihedral angles." multiple="true"/>
+            <param name="input_format" type="select" label="Input file format" help="Select the format of the input file or choose auto to let the system decide based on the file extension and content.">
+                <option value="auto" selected="true">Auto</option>
+                <option value="csv">CSV</option>
+                <option value="xvg">XVG</option>
+            </param>
+            <param name="input_degrees" type="select" label="Are the dihedral angles in that file in radians or degrees?" help="Indicate if the dihedral angles are in radians or degrees.">
+                <option value="Radians">Radians</option>
+                <option value="Degrees" selected="true">Degrees</option>
+            </param>
+        </section>
+       <section name="pdf_options" title="Kernel Options">
+            <conditional name="pdf">
+                <param name="use_pretrained_pdf" type="boolean" label="Do you want to use custom probability density functions (pdf)?"/>
```

Comment:
Instead of starting with a yes/no question, I think we should straight up give these options as optional. And make sure this is explained well in the help section. This is referring to the Planemo suggestion:
`WARNING (ConditionalParamTypeBool): Conditional [pdf] first param of type="boolean" is discouraged, use a select`

So in practice, that would look like this
![image](https://github.com/user-attachments/assets/83a4946c-30cb-434e-8573-e12d132f7fb1)

(The second yes/no should also be changed but don't have a proposal yet)
Instead of
![image](https://github.com/user-attachments/assets/c031e5ef-c84c-4e16-9b2b-0d311db0a5e7)